aspnetcore/mvc/views/overview.md

```diff
@@ -118,7 +118,16 @@ Follow the best practice of organizing the file structure for your views to refl
 
 ## Passing data to views
 
-You can pass data to views using several approaches. The most robust approach is to specify a [model](xref:mvc/models/model-binding) type in the view. This model is commonly referred to as a *viewmodel*. You pass an instance of the viewmodel type to the view from the action.
+Pass data to views using several approaches:
+
+* Strongly-typed data: viewmodel
```

Comment:
The hyphen isn't needed when following an "ly" word. Adjust below too.

Reply:
🤔 It's an compound modifier (compound adjective) here tho. As a noun, I would agree ... but as an adjective ... My rubber :duck: says 'hyphen' on this.